cas-server-support-ldap/src/main/java/org/apereo/cas/authentication/PooledCompareCustomAttributeAuthenticationHandler.java

```diff
@@ -0,0 +1,72 @@
+package org.apereo.cas.authentication;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+import org.ldaptive.CompareOperation;
+import org.ldaptive.CompareRequest;
+import org.ldaptive.Connection;
+import org.ldaptive.LdapAttribute;
+import org.ldaptive.LdapException;
+import org.ldaptive.LdapUtils;
+import org.ldaptive.Response;
+import org.ldaptive.auth.AuthenticationCriteria;
+import org.ldaptive.auth.AuthenticationHandlerResponse;
+import org.ldaptive.auth.PooledCompareAuthenticationHandler;
+import org.ldaptive.pool.PooledConnectionFactory;
+
+/**
+ * Extended ldaptive {@code Authenticator} component, which uses a custom ldap
+ * attribute for password comparison.
+ *
+ * @author Robert Kornmesser
+ * @since 5.0.0
+ */
+public class PooledCompareCustomAttributeAuthenticationHandler extends PooledCompareAuthenticationHandler {
```

Comment:
Did not have closer look on ldaptive. Maybe that is already integrated in a higher version, which we dont use yet. I see a PR for ldaptive 1.2. Maybe we should update ldaptive before?